File: examples/python/tasks/audio_classification/audio_classification.py
import requests
import base64
from bytez import Bytez


def get_base64_audio(url):
    response = requests.get(url)
    return base64.b64encode(response.content).decode("utf-8")


input_audio_base64 = get_base64_audio(
    "https://huggingface.co/datasets/huggingfacejs/tasks/resolve/main/audio-classification/audio.wav"
)

client = Bytez("YOUR BYTEZ KEY HERE")

model = client.model("aaraki/wav2vec2-base-finetuned-ks")

model.load()

result = model.run({"b64AudioBufferWav": input_audio_base64})

label_objects = result["output"]

for label_object in label_objects:
    # depending on the model, there may be additional props returned
    print(label_object)

    score = label_object["score"]
    label = label_object["label"]

    print({"score": score, "label": label})

File: examples/julia/tasks/audio_classification/audio_classification.jl
using Bytez
using Base64
using HTTP


# Function to get the audio as base64-encoded string
function get_base64_audio(url::String)::String
	response = HTTP.get(url)
	return base64encode(response.body)
end

# Get the base64 encoded audio from the URL
input_audio_base64 = get_base64_audio(
	"https://huggingface.co/datasets/huggingfacejs/tasks/resolve/main/audio-classification/audio.wav",
)

# Initialize the Bytez client with your API key
client = Bytez.init("YOUR BYTEZ KEY HERE")

# Load the model
model = client.model("aaraki/wav2vec2-base-finetuned-ks")

model.load()

# Run inference on the model
result = model.run(Dict("b64AudioBufferWav" => input_audio_base64))

# Process and print the results
label_objects = result["output"]

for label_object in label_objects
	score = label_object["score"]
	label = label_object["label"]
	println("Score: $score, Label: $label")
end
